src/components/ui/Chat/Message/Message.stories.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import { useTheme } from 'styled-components';
+import { select, boolean } from '@storybook/addon-knobs';
+
+import Flex from '../../Flex';
+import Message from './';
+import MessageDocs from './Message.mdx';
+import { PopOverItemProps } from '../../PopOver/PopOverItem';
+
+export default {
+  title: 'UI Components/Chat',
```

Comment:
Will this render as UI Components > Chat > Message?

Dave mentioned this in the past, but we probably want to group our components and documentation as Chat vs Meeting

Reply:
Yeah, it will follow that folder pattern. That's what we want, right?

Reply:
Yeah, I think grouping them by purpose should be good. Eventually we should probably do the same for a /meetings directory too